src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -40,25 +40,27 @@ google.auth.getApplicationDefault(function(err, authClient) {
     auth: authClient
   };
 
-  var recur = function(err, response) {
+  logging.entries.list(request, function handlePage(err, response) {
```

Comment:
Nope, its scoped like any other variable.

On Aug 21, 2016 12:59 AM, "Michael Darakananda" <notifications@github.com>
wrote:

> In src/test/java/com/google/api/codegen/testdata/discoveries/
> nodejs/nodejs_logging.v2beta1.json.baseline
> <https://github.com/googleapis/toolkit/pull/398#discussion_r75590564>:
>
> > @@ -40,25 +40,27 @@ google.auth.getApplicationDefault(function(err, authClient) {
> >      auth: authClient
> >    };
> >
> > -  var recur = function(err, response) {
> > +  logging.entries.list(request, function handlePage(err, response) {
>
> Does function handlePage pollute the global namespace?
> If I remember correctly, function foo puts foo into global namespace
> where var foo = function puts foo in the smallest scope like other
> variable declarations.
> In the sample, I don't think it greatly matters either way though.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/googleapis/toolkit/pull/398/files/caf14dc4c31ee98a5c2a84c8b388cce16077c371..a5a4a1904793f74af42572d73aef277a38e71dfb#r75590564>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABhljLvv6ZeaD38jTLjZyCwazi8MmSEks5qiAVCgaJpZM4JoNvK>
> .
>